Lib/fontTools/ttLib/tables/C_P_A_L_.py

```diff
@@ -236,12 +236,12 @@ def fromXML(self, name, attrs, content, ttFont):
 
 class Color(namedtuple("Color", "blue green red alpha")):
 
-	def hex(self):
-		return "#%02X%02X%02X%02X" % (self.red, self.green, self.blue, self.alpha)
-
 	def __repr__(self):
 		return self.hex()
 
+	def hex(self):
+		return "#%02X%02X%02X%02X" % (self.red, self.green, self.blue, self.alpha)
+
```

Comment:
please undo this change (moving `hex` method) as it is just diff noise, thanks

Reply:
ye, I am learning this culture of contributing. I saw it as very small improvement in legibility

Reply:
It changed the object type to no longer be a tuple subclass, so it would not be backwards compatible.

Reply:
that change was undone, it's still a namedtuple

Reply:
I know, I was just in an explainy mood...

Reply:
oh ok, thank you! :)